exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var greeting = "Hello ";
+var name1 = "my name is Maryam.";
```

Comment:
The space after full stop is not normally inserted unless you intend to add another string/number/variable to it to form a longer string.